scripts/redis_crash_handler.py

```
@@ -0,0 +1,1098 @@
+        pod = args.pod
+        namespace = args.namespace
+        cluster = args.cluster
+        
+        # Get Google Chat webhook early for error notifications
+        google_chat_webhook = os.environ.get('GOOGLE_CHAT_WEBHOOK_URL')
+        
+        # Configure SSL verification
+        verify_ssl = os.environ.get('DISABLE_SSL_VERIFY', 'false').lower() != 'true'
+        
+        main()
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)